Admin/AbstractAdmin.php

```diff
@@ -1202,9 +1203,20 @@ public function attachAdminClass(FieldDescriptionInterface $fieldDescription)
         $adminCode = $fieldDescription->getOption('admin_code');
 
         if ($adminCode !== null) {
-            $admin = $pool->getAdminByAdminCode($adminCode);
+            try {
+                $admin = $pool->getAdminByAdminCode($adminCode);
+            } catch (ServiceNotFoundException $e) {
+                throw new \RuntimeException('Unable to find the Admin instance', $e->getCode(), $e);
```

Comment:
```diff
- Unable to find the Admin instance
+ Unable to find the admin instance
```

"instance" -> "class" ??

Reply:
Instance is right, because you could instantiate a class several times.